### DISCUSSION TITLE
API stats for a goal

### DISCUSSION THREAD

**Discussion by glatzl** [plausible/analytics#1617] [Anything Else]
Hello! 

Would be great to be able to fetch stats for a specific goal (stats API).

**Comment by metmarkosaric** [comment]
hi! you should be able to do that already with event:name. see https://plausible.io/docs/stats-api#properties

**Comment by glatzl** [comment]
Thanks for the reply!
I can only find "Breakdown custom event by custom props"
If I only need to fetch a single event that doesn't have any custom props, how would I do that?

Closest I got for now is an array with all my goals
`https://plausible.io/api/v1/stats/breakdown?site_id=${SITE_ID}&property=event:goal`

**Reply by RobertJoonas** [reply]
Hey @glatzl! If you just want the number of conversions for a goal, you should use the aggregate endpoint instead:
`https://plausible.io/api/v1/stats/aggregate?site_id=${SITE_ID}&filters=event:name==YourGoalName`

**Reply by glatzl** [reply]
Hey @RobertJoonas. Thank you, this works perfectly!
